api/app/Models/PoolCandidate.php

```diff
@@ -250,9 +252,14 @@ public static function scopeExpiryStatus(Builder $query, ?string $expiryStatus)
         return $query;
     }
 
-    public function scopeNotDraft(Builder $query): Builder
+    public function scopeNotDraft(Builder $query, $where = [], bool $notDraft = true): Builder
```

Comment:
I thought scopes could have at most 2 arguments. What's the $where arg for?

Reply:
For some odd reason, it passes the `$where` arg from `Query.searchPoolCandidates`. I have no idea why but adding in the `$where = []` makes it work. If you `Log::debug($where);` it actually shows the object passed to it 😕

Reply:
Actually, turns out I was using the scope incorrectly and didn't need to pass any arguments. I defined it as a [local scope](https://lighthouse-php.com/5/eloquent/getting-started.html#local-scopes) but read the documentation for a[ regular scope](https://lighthouse-php.com/5/api-reference/directives.html#scope) 🤦‍♀️

Reply:
Ohhhh!